bowseries.go

```diff
@@ -125,6 +130,19 @@ func NewSeriesFromInterfaces(name string, typ Type, data []interface{}) Series {
 			builder.Append(v)
 		}
 		return Series{Name: name, Array: builder.NewArray()}
+	case TimestampSec, TimestampMilli, TimestampMicro, TimestampNano:
+		builder := array.NewTimestampBuilder(mem, mapBowToArrowTypes[typ].(*arrow.TimestampType))
```

Comment:
Could we extract this code block on function(s) ?

Reply:
We could, however I don't think this would improve anything here. The blocks are quite small.